pyvrp/cpp/Route.h

```diff
@@ -35,7 +79,6 @@ class Route
     Duration timeWarp_ = 0;        // Total time warp on this route
     Duration travel_ = 0;          // Total *travel* duration on this route
     Duration service_ = 0;         // Total *service* duration on this route
-    Duration wait_ = 0;            // Total *waiting* duration on this route
```

Comment:
This field wasn't actually used. Not sure why we even had this to begin with - it's trivially computable (and, that's indeed what happens when using `wait_duration()`).